ospd/misc.py

```diff
@@ -158,18 +164,27 @@ def get_hosts_unfinished(self, scan_id: str) -> List[Any]:
     def get_hosts_finished(self, scan_id: str) -> List:
         """ Get a list of finished hosts."""
 
-        finished_hosts = list() # type: List
+        finished_hosts = list()  # type: List
         for target in self.scans_table[scan_id]['finished_hosts']:
             finished_hosts.extend(
                 self.scans_table[scan_id]['finished_hosts'].get(target)
             )
 
         return finished_hosts
 
-    def results_iterator(self, scan_id: str, pop_res: bool) -> Iterator[Any]:
+    def results_iterator(
+        self, scan_id: str, pop_res: bool, max_res: int
```

Comment:
```suggestion
        self, scan_id: str, pop_res: bool, *, max_res: int = None
```
or even better but might require additional code changes
```python3
        self, scan_id: str, *, pop_res: bool = False, max_res: int = None
```